### PR TITLE
Shopify: add visual changes from ui library

### PR DESCRIPTION
**Pull request by cbernat**
Add visual styles from ui library for blocks only. 
### Connected:
image

### Not connected:
image

### With error:
image

### CODE REVIEW

File: src/components/Integrations/Shopify/Shopify.js [hunk 1]
```diff
@@ -30,29 +30,120 @@ const Shopify = ({ dependencies: { shopifyClient } }) => {
...
+      <hr />
+    </>
+  );
+
   return (
     <>
       <Helmet title={'Doppler | Shopify'} />
-      {/* inline style will be removed in next PR */}
-      <section style={{ width: '100%', padding: '60px 30px' }}>
+      {/* inline styles will be removed when breadcrum is ready in ui library */}
```
Comment (fcoronelmakingsense) [inline src/components/Integrations/Shopify/Shopify.js]: 👍

File: src/components/Integrations/Shopify/Shopify.js [hunk 2]
```diff
@@ -30,29 +30,120 @@ const Shopify = ({ dependencies: { shopifyClient } }) => {
...
   }, [shopifyClient]);
 
+  const shopifyHeader = (
+    <>
+      <div className="block">
+        <h2>Panel de Control - Shopify</h2>
+        <p>
+          Lorem ipsum, dolor sit amet consectetur adipisicing elit. Consequuntur facere velit
+          officia quibusdam perspiciatis impedit veritatis et, ducimus dolorem{' '}
+          <a href="http://fromdoppler.com">corporis</a>.
```
Comment (fcoronelmakingsense) [inline src/components/Integrations/Shopify/Shopify.js]: dont need link in english too?

File: src/components/Integrations/Shopify/Shopify.js [hunk 3]
```diff
@@ -30,29 +30,120 @@ const Shopify = ({ dependencies: { shopifyClient } }) => {
...
-        ) : (
-          <p>Shopify desconectado</p>
-        )}
+        <div className="dp-integration" style={{ marginTop: '20px' }}>
+          {isLoading ? (
+            <>
+              <div className="dp-integration__block">
+                {shopifyHeader}
+                <div className="block">
+                  <p></p>
```
Comment (fcoronelmakingsense) [inline src/components/Integrations/Shopify/Shopify.js]: empty <p>??

File: src/components/Integrations/Shopify/Shopify.js [hunk 4]
```diff
@@ -30,29 +30,120 @@ const Shopify = ({ dependencies: { shopifyClient } }) => {
...
+              <div className="dp-integration__block dp-integration--info">
+                <header className="block">
+                  <div>
+                    <h2>Listas creadas</h2>
+                    <p>
+                      Lorem ipsum, dolor sit amet consectetur adipisicing elit. Et praesentium
+                      distinctio obcaecati.
+                    </p>
+                  </div>
+                  {/*<button className="dp-button button--has-icon"><span class="ms-icon icon-reload"></span></button>*/}
```
Comment (fcoronelmakingsense) [inline src/components/Integrations/Shopify/Shopify.js]: need this?

File: src/components/Integrations/Shopify/Shopify.js [hunk 5]
```diff
@@ -30,29 +30,120 @@ const Shopify = ({ dependencies: { shopifyClient } }) => {
...
+                </header>
+                <hr />
+                <div className="block">
+                  <ul>
+                    {shops.map((shop) => (
+                      <li key={shop.list.id}>
+                        {' '}
```
Comment (fcoronelmakingsense) [inline src/components/Integrations/Shopify/Shopify.js]: need this space? cant use in `<p>` styles?

**Comment by andresmoschini** [comment]
👍

File: src/components/Integrations/Shopify/Shopify.js (hunk 2 above)
Comment (cbernat) [reply src/components/Integrations/Shopify/Shopify.js]: I have not language yet. This is only visual styles.

File: src/components/Integrations/Shopify/Shopify.js (hunk 3 above)
Comment (cbernat) [reply src/components/Integrations/Shopify/Shopify.js]: Oops! good catch!

File: src/components/Integrations/Shopify/Shopify.js (hunk 4 above)
Comment (cbernat) [reply src/components/Integrations/Shopify/Shopify.js]: It is a reminder, this is the button for sincronize, I don't know when we will do this. I prefer to keep this hidden.  Maybe I can add a comment about this.

File: src/components/Integrations/Shopify/Shopify.js (hunk 5 above)
Comment (cbernat) [reply src/components/Integrations/Shopify/Shopify.js]: Yes I will try to remo this, is very weird. O_o

File: src/components/Integrations/Shopify/Shopify.js (hunk 4 above)
Comment (cbernat) [reply src/components/Integrations/Shopify/Shopify.js]: @fcoronelmakingsense  added a todo here

**Comment by cbernat** [comment]
Test this please.

**Pull request merged**